src/vaadin-avatar.html

```diff
@@ -206,10 +211,6 @@
           this.__iconVisible = !this.img && !this.abbr;
         }
 
-        __nameChanged(name) {
-          this.__setTitleFromName(name);
-        }
-
         __setTitleFromName(name) {
```

Comment:
nit: Does it make sense now to rename the method, i.e. to `__setTitle(string)`? 🤔

Reply:
Thanks, done.